ground/gcs/src/plugins/uploader/bl_messages.h

```diff
@@ -85,6 +85,103 @@ enum bl_commands {
  * Note: These enum values MUST NOT be changed or backward
  *       compatibility will be broken
  */
+#ifdef _MSC_VER
```

Comment:
We need to figure out how to get rid of all this duplication.  Perhaps factoring out the union declarations to be outside the struct would help.